src/Mpdf.php

```diff
@@ -1579,24 +1576,6 @@ public function cleanup()
 		$this->createdReaders = [];
 	}
 
-	/**
```

Comment:
I gather that this functionality would be lost then?

Reply:
Oh. I don't know I'd want this for all classes using the trait, splitting this to "simple" and "mpdf-only" traits seems reasonable.

Reply:
Since this is your main application class, maybe the initialization of the service loggers could simply be moved to another method. It wouldn't be good to just leave the services uninitialized; I suspect this was just an oversight. (I didn't notice it, on first review.)

Reply:
I see the issue now; you want to allow anyone with a reference to the Mpdf application to be able to call `setLogger` on it, and have the logger be applied to all of the Mpdf services. Having a specific mpdf-only trait (or perhaps a "service container" trait) would be a reasonable way to restore this functionality.